backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/starrating/BpkStarRating.kt

```diff
@@ -81,13 +81,31 @@ fun BpkStarRating(
     )
 }
 
+@OptIn(ExperimentalBackpackApi::class)
 @Composable
 fun BpkHotelRating(
     rating: Int,
     contentDescription: ContentDescriptionScope.(Float, Int) -> String,
     modifier: Modifier = Modifier,
     size: BpkStarRatingSize = BpkStarRatingSize.Small,
-    color: BpkRatingColor? = BpkRatingColor.Yellow,
+) {
+    BpkHotelRating(
+        rating = rating,
+        contentDescription = contentDescription,
+        modifier = modifier,
+        size = size,
+        color = BpkRatingColor.Yellow,
+    )
+}
+
+@OptIn(ExperimentalBackpackApi::class)
```

Comment:
created a separate function for the one with the rating prop otherwise it'll trigger on any usage of `BpkHotelRating`